README.md

```diff
@@ -212,6 +216,40 @@ OPTIONS:
     -h, --help               Print help information
 ```
 
+#### dev
+
+This subcommand starts the Lenra app of the current directory in dev mode.
+
+```bash
+$ lenra dev --help
+lenra-dev 
+Starts the dev mode
+
+USAGE:
+    lenra dev [OPTIONS]
+
+OPTIONS:
+        --config <CONFIG>    The app configuration file [default: lenra.yml]
+    -h, --help               Print help information
+```
+
+#### init
+
+This subcommand generates Docker and Docker Compose files.
```

Comment:
```suggestion
This subcommand generates Dockerfile and Docker Compose files.
```

Reply:
It not just the `Dockerfile`, but also the `.dockerignore`